convert.c

```diff
@@ -899,7 +899,7 @@ static int apply_multi_file_filter(const char *path, const char *src, size_t len
 	if (fd >= 0)
 		err = write_packetized_from_fd(fd, process->in);
 	else
-		err = write_packetized_from_buf(src, len, process->in);
+		err = write_packetized_from_buf(src, len, process->in, 1);
```

Comment:
There's an odd asymmetry about this.  We only have 1 caller of `_from_buf()` and `_from_fd()`, yet only one gets the 3rd argument.

I'm wondering if we should update both to have consistent behavior -- OR -- move the flush to here following the call (when !err).

Reply:
I did not want to introduce unneeded code changes. Currently, I do not need the `_from_fd()` function, and it is not even clear whether there will ever be a use case where `_from_fd()` needs to _not_ send a flush packet at the end.

Does that make sense?